src/library/Shared/Implementations/ExtendedSplashScreen.Android.cs

```diff
@@ -60,19 +61,29 @@ private static void OnExitAnimation(object sender, AndroidX.Core.SplashScreen.Sp
 		{
 			Trace("Started OnExitAnimation.");
```

Comment:
Does this actually follow the appropriate log logic here?

Reply:
All logs are traces in this class.